test/SVMSpokePoolClient.fills.ts

```diff
@@ -60,11 +64,11 @@ describe("SVMSpokePoolClient: Fills", function () {
     inputToken = EvmAddress.from(randomAddress());
     depositId = getRandomInt();
     relayData = {
-      depositor,
-      recipient,
-      exclusiveRelayer: toAddressType(SVM_DEFAULT_ADDRESS, CHAIN_IDs.SOLANA),
-      inputToken,
-      outputToken: toAddressType(mint.address, CHAIN_IDs.SOLANA),
+      depositor: toAddress(depositor),
+      recipient: toAddress(recipient),
+      exclusiveRelayer: toAddress(SvmAddress.from(SVM_DEFAULT_ADDRESS)),
+      inputToken: toAddress(inputToken),
+      outputToken: toAddress(SvmAddress.from(mint.address)),
```

Comment:
This change (and most others in this file) are just to address type complaints. My editor lit up like a christmas tree in this file.